package.json

```diff
@@ -68,7 +68,7 @@
     "rebuild:electron": "theia rebuild:electron",
     "rebuild:electron:debug": "DEBUG=electron-rebuild && yarn rebuild:electron",
     "publish": "yarn && yarn test && yarn publish:latest",
-    "publish:latest": "lerna publish && yarn publish:check",
+    "publish:latest": "lerna publish --exact && yarn publish:check",
```

Comment:
@Niklas-Dahlquist I could not find any information for the `--exact` flag in the `lerna publish` documentation:
- https://github.com/lerna/lerna/tree/main/commands/publish

Can you perhaps include a link to documentation for the `--exact` flag in the commit message, and/or pull-request description?

Reply:
Hi Vince!

The information for --exact is hidden under lerna version, see this direct link

* https://github.com/lerna/lerna/tree/main/commands/version#--exact

/Niklas